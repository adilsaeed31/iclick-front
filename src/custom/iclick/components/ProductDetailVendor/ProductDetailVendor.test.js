import React from "react";
import renderer from "react-test-renderer";
import ProductDetailVendor from "./ProductDetailVendor";

test("basic snapshot", () => {
  const component = renderer.create((
    <ProductDetailVendor>Vendor</ProductDetailVendor>
  ));
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});

test("no vendor", () => {
  const component = renderer.create((
    <ProductDetailVendor />
  ));
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
