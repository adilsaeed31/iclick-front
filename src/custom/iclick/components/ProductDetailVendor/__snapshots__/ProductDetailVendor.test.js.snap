// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`basic snapshot 1`] = `
<div
  className="MuiTypography-root-1 MuiTypography-body1-10"
>
  Vendor
</div>
`;

exports[`no vendor 1`] = `null`;
